### PR TITLE
[knx] Improve config description for GAs

### PR DESCRIPTION
**Pull request by holgerfriedrich**
Carryover from smarthomej/addons#257.

Reference: #14502

Also-by: Jan N. Klug <github@klug.nrw>

### CODE REVIEW

File: bundles/org.openhab.binding.knx/src/main/resources/OH-INF/i18n/knx.properties
Comment (jlaur) [inline bundles/org.openhab.binding.knx/src/main/resources/OH-INF/i18n/knx.properties:148]: Moved unintentionally? Usually this is in the beginning of the file.

Comment (holgerfriedrich) [reply bundles/org.openhab.binding.knx/src/main/resources/OH-INF/i18n/knx.properties:148]: This is done by the i18n tooling when I regenerated the translations. I gave up to move it back to the beginning. Have you seen this in other binding as well or could it be related to my setup?

Comment (jlaur) [reply bundles/org.openhab.binding.knx/src/main/resources/OH-INF/i18n/knx.properties:148]: For the main branch, for me running `mvn i18n:generate-default-translations` gives this result:
image

Comment (holgerfriedrich) [reply bundles/org.openhab.binding.knx/src/main/resources/OH-INF/i18n/knx.properties]: @jlaur thank you for checking... I just had a look at my batch file... and found the "3.4.0" I had overlooked before. Now it works fine.

**Comment by holgerfriedrich** [comment]
@J-N-K @kaikreuzer can we merge this? It is carryover and seems to be forgotten....

**Review by J-N-K — APPROVED** [review]
LGTM

**Comment by J-N-K** [comment]
@holgerfriedrich Unfortunately I don't have merge rights at the moment.

**Comment by kaikreuzer** [comment]
@J-N-K Now you do. 😎

**Review by kaikreuzer — APPROVED** [review]
...but as it is an "also-by @J-N-K", let me merge it.

**Comment by holgerfriedrich** [comment]
@cweitkamp can I ask you to review/approve the new German translations (knx.properties)?
Do we have a process how to request approvals for new translations, or is this just done from time to time?

**Comment by holgerfriedrich** [comment]
@cweitkamp can I ask you to review/approve the new German translations (knx.properties)?
Do we have a process how to request approvals for new translations, or is this just done from time to time?

**Pull request merged**
